DS/DSassignment1/MyList.java

```diff
@@ -0,0 +1,136 @@
+import java.util.Scanner;
+
+public class MyList {
+	
+	
+	
+	public int index;
+	
+	Scanner scan = new Scanner(System.in);
+	
+	ArrayList1 arrayList1Object = new ArrayList1();
+	Object value;
+	
+	public static void main(String[] args)
+	{
+		MyList object = new MyList();
+		object.arrayOperations();
+	}
+		
+	public void arrayOperations()
```

Comment:
I don't think it is asked to have user inputs. Please don't waste time in such things as all these can be easily tested in a test class which by the way is missing :)